frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/coordsProperty/coordsProperty.helpers.component.tsx

```diff
@@ -0,0 +1,66 @@
+/**
+ *  Copyright (C) 2023 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { rgbaToHex } from '@/v4/helpers/colors';
+import { get, isArray, isObject } from 'lodash';
+import { theme } from '@/v5/ui/themes/theme';
+import { ITemplate } from '@/v5/store/tickets/tickets.types';
+import { contrastColor } from 'contrast-color';
+import { AdditionalProperties } from '../../../tickets.constants';
+
+const DEFAULT_COLOR = theme.palette.primary.main;
+
+const findByName = (array, name) => array.find(({ name: n }) => n === name);
+const rgbArrayToHex = (obj) => rgbaToHex(obj?.toString());
```

Comment:
check out `rgbToHex` in `colorPicker.helpers.ts`. Should do the same and does not rely on v4 logic. Maybe we can extract that logic into a general color.helper file